pennylane/qnode.py

```diff
@@ -482,6 +483,17 @@ def __init__(
         functools.update_wrapper(self, func)
         self._transform_program = qml.transforms.core.TransformProgram()
 
+    def __copy__(self):
+        copied_qnode = QNode.__new__(QNode)
+        for attr, value in vars(self).items():
+            if attr not in {"execute_kwargs", "_transform_program"}:

```

Comment:
Should you also add gradient_kwargs here?

Reply:
added. thanks for catching that.